api/server/server.go

```diff
@@ -36,18 +41,27 @@ var (
 
 // Server represents the configured lotus client and filecoin grpc server
 type Server struct {
-	ds   datastore.TxnDatastore
-	dm   *deals.Module
-	ai   *ask.AskIndex
-	wm   *wallet.Module
-	si   *slashing.SlashingIndex
-	mi   *miner.MinerIndex
+	ds datastore.TxnDatastore
+
 	ip2l *ip2location.IP2Location
 
-	rpc           *grpc.Server
-	dealsService  *deals.Service
-	walletService *wallet.Service
-	closeLotus    func()
+	ai *ask.AskIndex
+	mi *miner.MinerIndex
+	si *slashing.SlashingIndex
+	dm *deals.Module
+	wm *wallet.Module
+	rm *reputation.ReputationModule
+
+	dealsService      *deals.Service
+	walletService     *wallet.Service
+	reputationService *reputation.Service
+	askService        *ask.Service
+	minerService      *miner.Service
+	slashingService   *slashing.Service
+
+	rpc *grpc.Server
+
+	closeLotus func()
```

Comment:
Just did some organizing here and below to keep things visually clear and consistent.